lib/puppet/configurer.rb

```diff
@@ -293,11 +296,10 @@ def run_internal(options)
       end
 
       current_environment = Puppet.lookup(:current_environment)
-      local_node_environment =
       if current_environment.name == @environment.intern
-        current_environment
+        local_node_environment = current_environment
```

Comment:
Why change this?

Reply:
Because I found it confusing and hard to read. Happy to put it back, it doesn't affect the rest of the work.

Reply:
Ok. It's a pattern I use occasionally, but this one seemed poorly formatted.